openedx/core/djangoapps/content/course_overviews/models.py

```diff
@@ -21,6 +21,7 @@
 
 from ccx_keys.locator import CCXLocator
 
+THUMBNAIL_DIMENSIONS = (375, 200)
```

Comment:
FWIW, the vast majority of source course images on edx.org are <= 375 pixels wide. It's just that when we have outliers, we have insane outliers (e.g. 8854x4897)